apps/api/src/app/preferences/dtos/preferences.dto.ts

```diff
@@ -0,0 +1,42 @@
+import { Type } from 'class-transformer';
+import { IsBoolean, ValidateNested } from 'class-validator';
+
+class Preference {
```

Comment:
```suggestion
export class Preference {
```
We need to export all nested DTOs to ensure that `@nestjs/swagger` can access all the nested typings when it produces the OpenAPI spec.